software/usertour.yml

```
@@ -0,0 +1,10 @@
+name: "Usertour"
+website_url: "https://www.usertour.io/"
+description: "User onboarding platform allowing you to create in-app product tours, checklists, and surveys in minutes effortlessly and with full control."
```

This part is redundant, as this is list is only for such software.
```suggestion
description: "User onboarding platform allowing you to create in-app product tours, checklists, and surveys in minutes effortlessly."
```